### PR TITLE
adding changes to incr and decr of item within a cart

### PR DESCRIPTION
**Pull request by rajaranjanmr**
PR for incr and decr of item within a cart

### CODE REVIEW

File: src/utility/cart-function.js [hunk 1]
```diff
@@ -20,23 +20,37 @@ const removeFromCart = async (item, setCart) => {
...
+    headers: { authorization: encodedToken },
+    data: {
+      action:{
+        type:'decrement'
+      }
+    },
+    
+  });
+    
+  console.log('from incre',response)
```
Comment (sri-rishi) [inline src/utility/cart-function.js]: Suggestion: just a remainder, remove `console` from code

**Review by sri-rishi — APPROVED** [review]
Code looks good , feel free to merge

File: src/utility/cart-function.js [hunk 2]
```diff
@@ -20,23 +20,37 @@ const removeFromCart = async (item, setCart) => {
...
+    headers: { authorization: encodedToken },
+    data: {
+      action:{
+        type:'increment'
+      }
+    },
+    
+  });
+    
+  console.log('from incre',response)
```
Comment (nikhiljugale007) [inline src/utility/cart-function.js]: remove console.log

File: src/components/cart-listing.js [hunk 3]
```diff
@@ -34,15 +28,15 @@ function CartListing({ value }) {
       <div className="quantity-up-down">
         Quantity :
         <p>
-          <i className="fa fa-plus" onClick={() => incrementClickHandler()}></i>
+          <i className="fa fa-plus" onClick={() => incrementClickHandler(value)}></i>
         </p>
-        <p style={{ border: "1px solid black", display: "inline" }}>
+        <p style={{ border: "1px solid black", display: "inline" }} >
```
Comment (nikhiljugale007) [inline src/components/cart-listing.js]: try not to use inline styling.

**Review by nikhiljugale007 — COMMENTED** [review]
Did not found any major issue.
Good to merge.

**Pull request merged**